### ISSUE TITLE
Mulitple extensible fields per object

### ISSUE THREAD

**Issue by JasonGlazer** [pyenergyplus/eppy3000#19]
I noticed you were working on this and just wanted to mention that one of the advantages of JSON is the idea of having multiple extensible fields for a given object since they are nested rather than at the end of an object.

**Comment by santoshphilip** [comment]
yup ! 
I agree. The eppy3000 can work with a lot of that. I wrote eppy3000 mostly as a proof of concept, but it is looking fairly robust. Lot of things that were hard with the IDD are trivially easy with the JSON.

most of the code I am writing for extensible fields are  in eppy or witheppy
https://github.com/pyenergyplus/witheppy/issues/13 and
https://github.com/santoshphilip/eppy/issues/238

I got your JSON presentation. Looking thru it now.
